docs/pallas.md

```diff
@@ -36,19 +36,42 @@ For simple kernels, the adoption is just as simple as one liner. For more compli
 
 ## Use built-in kernels
 
-Besides manually wrapping external Pallas kernels, there are built-in kernels where the adoptions are done by PyTorch/XLA already.
+Besides manually wrapping external Pallas kernels, there are built-in kernels where the adoptions are done by PyTorch/XLA already. These built-in kernels can be used like any other torch.ops. The current built-in kernels that are suppored are:
+- FlashAttention
+- PagedAttention
 
-Example usage:
+### FlashAttention
+
+#### Example usage
 ```python3
 # Use built-in kernels
 from torch_xla.experimental.custom_kernel import flash_attention
 output = flash_attention(q, k, v)
 ```
 
-You can just use it like any other torch.ops.
+#### Integration Example
+We have a fork of HuggingFace Llama 3 to demonstrate [a potential integration for FlashAttention here](https://github.com/pytorch-tpu/transformers/tree/alanwaketan/flash_attention).
+
+### PagedAttention
+
+#### Example usage
+```python3
+# Use built-in kernels
+from torch_xla.experimental.custom_kernel import paged_attention
```

Comment:
can you just do `import torch_xla.experimental.custom_kernel`, I will soon move the paged attention to https://github.com/pytorch/xla/blob/master/torch_xla/core/custom_kernel.py which will be auto-registered

Reply:
Updated